external/derby/test.properties

```diff
@@ -3,7 +3,4 @@ script="derby-test.sh"
 home_path="\${WORKDIR}/derby"
 tag_version="trunk"
 ant_version="1.10.6"
-debian_packages="git wget tar"
-alpine_packages="bash git wget tar"
-centos_packages="git wget tar"
-ubi_packages="git wget tar"
+ubuntu_packages="git maven"
```

Comment:
ubuntu_packages should be same as debian_packages. In this case `ubuntu_packages="git wget tar"`. Same for others.

Reply:
thank you @sophia-guo for the review. does that neccesitate following the debian_packages logic as used